### PR TITLE
Cleaned up some English in the documentation

### LINKED ISSUE #104: Bad English in the README
The README and other documentation files contain some bad English. It would be good to clean that up.

### PR DESCRIPTION
**Pull request by jarimayenburg**
I really like your tool but I found the README quite hard to follow because of the language mistakes. While I'm not a native English speaker myself, I tried my best to clean it up a little.

Closes #104.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,10 +131,9 @@ While you are welcome to provide your own configuration, Killgrave will follow t
 * **proxy**: `none`
 * **watcher**: `false`
 
-### Using Killgrave by command line
+### Using Killgrave in the command line
```
Comment (jarimayenburg) [inline README.md:134]:
```suggestion
### Using Killgrave from the command line
```

File: README.md [hunk 2]
```diff
@@ -117,13 +116,13 @@ Windows and Linux users can download binaries from the [Github Releases](https:/
...
 
-The very easy way to start with Killgrave is just execute at it is:
+To start Killgrave, you simply run the following.
 
 ```sh
 $ killgrave
 ```
 
-While you are welcome to provide your own configuration, Killgrave will follow the next default configuration:
+While you are welcome to provide your own configuration, Killgrave will default to the followin configuration:
```
Comment (jarimayenburg) [inline README.md:125]:
```suggestion
While you are welcome to provide your own configuration, Killgrave will default to the following configuration:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,40 +1,36 @@
...
 ## Questions
 
-If you have questions, or you want to share knowledge about Killgrave, feel free to join our [#killgrave slack channel](https://gophers.slack.com/archives/C01CJCP04E8).
+If you have questions, or if you want to share knowledge about Killgrave, feel free to join our [#killgrave slack channel](https://gophers.slack.com/archives/C01CJCP04E8).
 Also, we share announcements related with the tool on this channel.
 
 ## Getting Started
 
-### Filling a bug or feature 
+### Filing a bug or feature
```
Comment (jarimayenburg) [inline CONTRIBUTING.md:17]:
```suggestion
### Filing a bug report or feature request
```

File: README.md [hunk 4]
```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
...
 Killgrave provides:
 
-* Easy way to create imposters files, using `json`
-* The possibility to validate your owns [json schemas](https://json-schema.org/) on requests.
-* Validations the requests headers.
-* Using regex to allow different parameters on headers and urls.
-* Custom body and dynamic body responses
+* An easy way to create imposters files, using `json`
+* The possibility to validate requests against [json schemas](https://json-schema.org/).
+* Validation of requests headers.
```
Comment (jarimayenburg) [inline README.md:53]:
```suggestion
* Validation of request headers.
```

File: README.md [hunk 5]
```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
...
 
 ### Imposters
-Maybe the most common concept inside of Killgrave tool is the `imposters`. Imposters are the files that we will use to configure
-how we want that our server respond.
 
-You can identify an `imposter` file on Killgrave because they must have the extension `.imp.json`.
+Imposters are the most important concept of the Killgrave tool. They define the rules that determine how the server should respond to a request.
 
-Yo can learn more about how to configure the imposters on the [Imposter Configuration Section](#imposter).
+You can identify a Killgrave imposter file by their extension: `.imp.json`.
```
Comment (jarimayenburg) [inline README.md:70]:
```suggestion
You can identify a Killgrave imposter file by its extension: `.imp.json`.
```

File: README.md [hunk 6]
```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
...
-You can install Killgrave on different ways, but all of them are very simple:
+You can install Killgrave in different ways, but all of them are very simple:
 
 ### Go Toolchain
 
-One of them is of course using `go get`, Killgrave is a Go project so could be compiled using the `go toolchain`:
+One of them is of course using `go get`, Killgrave is a Go project and can therefore be compiled using the `go toolchain`:
 
 ```sh
 $ go get -u github.com/friendsofgo/killgrave/cmd/killgrave@{version}
```
Comment (joanlopez) [inline README.md:84]: Could be interesting to take the opportunity of this clean up to mention that since Go v1.16 it's better to use `go install` instead of `go get` for installing binaries from source code.

What do you think about simply swapping both commands? 
I think latest versions of Killgrave require Go v1.16 (because we started to use `goembed`), so I think we can assume the user should have Go v1.16+ installed if wants to install Killgrave from source code.

File: README.md [hunk 7]
```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
...
 ```
 
-`version` must be substituted by the `version` that you want to install, otherwise `main` would be installed.
+`version` must be substituted by the `version` that you want to install. If left unspecified, the `main` branch will be installed.
 
 ### Homebrew 
 
-If you are a mac user, you could install Killgrave using, [Homebrew](https://brew.sh/):
+If you are a Mac user, you can install Killgrave using, [Homebrew](https://brew.sh/):
```
Comment (joanlopez) [inline README.md:91]:
```suggestion
If you are a Mac user, you can install Killgrave using [Homebrew](https://brew.sh/):
```

Already existing extra comma that could be removed? 🤔

File: README.md [hunk 8]
```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
...
 
 ```sh
 $ brew install friendsofgo/tap/killgrave
 ```
 
-:warning:  If you are installing via homebrew, you only get the [last Killgrave version](https://github.com/friendsofgo/killgrave/releases), we hope fix this soon.
+:warning:  If you are installing via homebrew, you always get the [latest Killgrave version](https://github.com/friendsofgo/killgrave/releases), we hope fix this soon.
```
Comment (joanlopez) [inline README.md:97]:
```suggestion
:warning:  If you are installing via Homebrew, you always get the [latest Killgrave version](https://github.com/friendsofgo/killgrave/releases), we hope fix this soon.
```

File: README.md [hunk 9]
```diff
@@ -200,74 +196,68 @@ mymock/
...
-The option `proxy` allow to configure the mock on a proxy mode, that means that you could configure an fallback urls, for all
-your calls or only the missing ones or none. More information: [Proxy Section](#prepare-killgrave-for-proxy-mode)
+The option `proxy` allows you to configure the mock in proxy mode. When this mode is enabled, Killgrave will forward any unconfigured requests to another server. More information: [Proxy Section](#prepare-killgrave-for-proxy-mode)
 
 ## How to use
 
 ### Configure CORS
 
-If you want to use `killgrave` on your client application you must consider to configure correctly all about CORS, thus we offer the possibility to configure it as you need through a config file.
+If you want to use `killgrave` in a client application you should consider configuring CORS.
```
Comment (joanlopez) [inline README.md:217]:
```suggestion
If you want to use `killgrave` from a client application you should consider configuring CORS.
```

"From" sounds a bit better for me because if you're testing a frontend application, you consume Killgrave exposed (mocked) API but not within the client itself (as a dep or something like that).

File: README.md [hunk 10]
```diff
@@ -200,74 +196,68 @@ mymock/
...
+* `all`: The mock server will always call to the real server, declared in the `proxy-url` configuration variable.
 
-### Create an Imposter
+The `proxy-url` must be the root path of the proxied server. For example, if we have an API running on `http://example.com/things`, the `proxy-url` will be `http://example.com`.
 
-You must be creating an imposter to start using the application, only files with the `.imp.json` extension will be interpreted as imposter files, and the base path for the rest of the files will be the path of the `.imp.json` file.
+### Creating an Imposter
 
-You need to organize your imposters from more restrictive to less. We use a rule-based system for creating each imposter, for this reason you need to organize your imposters from the most restrictive to the least, like the example below.
+At least one imposter must to be configured in order to run Killgrave. Files with the `.imp.json` extension in the `imposters` folder (default "imposters") will be interpreted as imposter files.
```
Comment (joanlopez) [inline README.md:258]:
```suggestion
At least one imposter must be configured in order to run Killgrave. Files with the `.imp.json` extension in the `imposters` folder (default "imposters") will be interpreted as imposter files.
```

File: README.md [hunk 11]
```diff
@@ -287,62 +277,48 @@ imposter example:
...
+* `endpoint` (<span style="color:red">mandatory</span>): Path of the endpoint relative to the base. Supports regex.
+* `schemaFile`: A JSON schema to validate the incoming request against.
+* `params`: Restrict incoming requests by query parameters. More info can be found [here](#create-an-imposter-with-query-params). Supports regex.
+* `headers`: Restrict incoming requests by HTTP header. More info can be found [here](#create-an-imposter-with-headers).
 
 #### Response
 
-On the other hand we have the `response` object, the `response` object will be in charge to generate the output that
-we want, related with the `request` that was executed.
+This part defines how Killgrave should respond to the incoming request. The `response` object hsa the following properties:
```
Comment (joanlopez) [inline README.md:303]:
```suggestion
This part defines how Killgrave should respond to the incoming request. The `response` object has the following properties:
```

File: README.md [hunk 12]
```diff
@@ -489,7 +461,7 @@ So with this `json schema`, we expect a `request` like this:
 }
 ```
 
-Then our `imposter` will be configure in the next way:
+Then our imposter will be configure as follows:
```
Comment (joanlopez) [inline README.md:464]:
```suggestion
Then our imposter will be configured as follows:
```

**Review by joanlopez — CHANGES_REQUESTED** [review]
Heyaaa @jarimayenburg, thanks so much for your contributions!! 🙏🏻 
I left few comments/suggestions, could you review them, please?
Apart from those minor comments, it looks super good overall 😊

File: README.md (hunk 6 above)
Comment (jarimayenburg) [reply README.md]: Sounds good! I updated it.

**Comment by jarimayenburg** [comment]
@joanlopez Any indication of when this can be merged?

**Review by joanlopez — APPROVED** [review]
Looks great! Thanks so much! 🙏🏻 👌🏻

PS: This probably needs to be backported to v0.5.0 as well - cc/ @aperezg

**Comment by joanlopez** [comment]
> @joanlopez Any indication of when this can be merged?

You have my 👍🏻  So as soon as @aperezg has a quick look and approves it. Thanks! 🙏🏻

**Comment by aperezg** [comment]
LGTM!! Thanks a lot for the contribution!!

**Review by aperezg — APPROVED** [review]

**Comment by jarimayenburg** [comment]
Great! Thanks for the reviews! Ready to merge?
